hack/lib/OWNERS

```diff
@@ -0,0 +1,13 @@
+reviewers:
+  - stevekuznetsov
+  - smarterclayton
+  - kargakis
+  - soltysh
+  - liggitt
+  - jhadvig
+  - fabianofranz
```

Comment:
this owners file looks wrong/out of date.

Reply:
removed.